Sources/ComposableArchitecture/TestSupport/TestStore.swift

```diff
@@ -235,6 +235,9 @@
         },
         environment: ()
       )
+      #if DEBUG
+      store.stackChecksEnabled = false
```

Comment:
Technically, `TestStore` can still emit a warning when it initializes. The workaround would be to make the `Store.init` that expose checks flags internal instead of private. This would also allow to remove this line.